data/libs/Character.lua

```diff
@@ -358,16 +222,8 @@ Character = {
 --
 -- Integer attribute for RPG style dice tests.  Piloting is intended to reflect
 -- the character's skill at flying spacecraft.
--- Tested with 4xD16; useful values are 4 (always crashes) to 65 (aerobat).
+-- Tested with 4xD16; useful values are 4 (always crashes) to 65 (acrobat).
```

Comment:
I don't think this is necessarily misspelled? https://en.wikipedia.org/wiki/Aerobatics

Reply:
My google is weak....